labs/Language/Basics.md

```diff
@@ -216,6 +216,11 @@ import labs::ProjectEuler::ProjectEuler as PE hiding (inc, inc1001)
 Cryptol's module system also supports parameters, but that is covered
 in a later lab.
 
+### File-only commands
+
+It's worth noting that there are a very few Cryptol commands
+that can only be used in a file, *not* interactively in the interpreter.  The most common of these are `module` and `import`.  
+
```

Comment:
Well, why not just add `private` and `property` to the list?

Reply:
Sure.  Are they that common?

Reply:
Yes. Especially `property`. It shows up in _almost_ every lab.